wolfssl/wolfcrypt/aes.h

```diff
@@ -209,6 +205,9 @@ struct Aes {
     word32 y0;
 #endif
 #endif /* HAVE_AESGCM */
+#ifdef WOLFSSL_CAAM
```

Comment:
pulled out of AES-GCM macro guard, it could be used with AES-CBC or other AES builds